docs/2-Presets/Wordpress.md

```diff
@@ -0,0 +1,48 @@
+### Start from Scratch
+
+For wordpress preset start from scratch or a existing wordpress is the same, go to the next section.
```

Comment:
If you want to create a project from scratch, just go to the directory you wanna have the new Wordpress install and run:

```console
$ kool init wordpress
$ kool start
```

Upon first start, if the current working directory is not an existing Wordpress source code, it will create it for you.